src/balancerd/src/lib.rs

```diff
@@ -18,6 +18,7 @@
 
 mod codec;
 mod dyncfgs;
+// mod service;
```

Comment:
I think you may have left this in by mistake.

Reply:
I totally did, thanks!